app/templates/location.html

```diff
@@ -14,15 +14,19 @@
     <div class="nav">
       <div class="nav">
         <ul>
-            <input type="text" placeholder="Search users">
+          <form action="/search-user" method="get" id="search-form">
+            <input class="search" type="text" name="username" placeholder="Enter Username">
+            <input class="search-btn" type="submit" value="search">
```

Comment:
@azaria-dedmon `search` should be capitalized. Also make sure the search button is exactly the same width as the search field.